### ISSUE TITLE
Compatibility with Django 4.0

### ISSUE THREAD

**Issue by ronnievdc** [mxsasha/django-restricted-sessions#20]
Django restricted sessions still uses the `force_text` but has been removed in django 4.0

# Release notes Django 4.0

https://docs.djangoproject.com/en/4.0/releases/4.0/#features-removed-in-4-0
```
django.utils.encoding.force_text() and smart_text() are removed.
```

**Comment by ronnievdc** [comment]
I created a PR: https://github.com/mxsasha/django-restricted-sessions/pull/21

It was difficult to keep the old unmaintained Django versions because the Nose testrunner is not compatible with the new Python and Django versions. Therefore i went for only the supported versions of Django.

**Comment by mxsasha** [comment]
Fixed in 0.4.0. Thanks!
